### ISSUE TITLE
person_test.UT_IsSame.test_06_same_and_more_hub_adult FAILS

### ISSUE THREAD

**Issue by djfkahn** [djfkahn/MemberHubDirectoryTools#100]
```
WHERE:
	 test_06_same_and_more_hub_adult (person_test.UT_IsSame)
WHAT:
 Traceback (most recent call last):
  File "/Users/djfkahn/Desktop/DEV/MemberHubDirectoryTools/TEST/person_test.py", line 91, in test_06_same_and_more_hub_adult
    self.assertFalse(person_A.IsSame(person_B))
AssertionError: True is not false
```

**Comment by djfkahn** [comment]
This is related to issue #88.  Need to mark this as an expected Failure.

**Comment by djfkahn** [comment]
addressed in 50a7d96
